stackslib/src/chainstate/nakamoto/mod.rs

```diff
@@ -1743,17 +1832,36 @@ impl NakamotoChainState {
             return Ok(false);
         };
 
-        let schnorr_signature = &block.header.signer_signature.0;
-        if !db_handle.expects_signer_signature(
-            &block.header.consensus_hash,
-            schnorr_signature,
-            &block.header.signer_signature_hash().0,
-            aggregate_public_key,
-        )? {
-            let msg = format!(
-                "Received block, but the signer signature does not match the active stacking cycle"
+        // TODO: epoch gate to verify aggregate signature
+        // let schnorr_signature = &block.header.signer_signature.0;
+        // if !db_handle.expects_signer_signature(
+        //     &block.header.consensus_hash,
+        //     schnorr_signature,
+        //     &block.header.signer_signature_hash().0,
+        //     aggregate_public_key,
+        // )? {
+        //     let msg = format!(
+        //         "Received block, but the signer signature does not match the active stacking cycle"
+        //     );
+        //     warn!("{}", msg; "aggregate_key" => %aggregate_public_key);
+        //     return Err(ChainstateError::InvalidStacksBlock(msg));
+        // }
+
+        // TODO: epoch gate to verify signatures vec
```

Comment:
I'm not sure what you mean?  This code can't be reached for pre-3.0 blocks.

Reply:
Yeah, I basically mean "once we have epoch gates for threshold signatures, this should be behind an `epoch == 3.0` branch. I can remove though, it's implicit in the above comment

Reply:
Got it